src/Sulu/Bundle/AdminBundle/Admin/View/ListViewBuilderTrait.php

```diff
@@ -35,8 +35,39 @@ private function setTitleToView(View $route, string $title): void
         $route->setOption('title', $title);
     }
 
+    /**
+     * @param string[] $listAdapters
+     */
     private function addListAdaptersToView(View $route, array $listAdapters): void
     {
+        /**
+         * @deprecated this is only a BC-layer, should be removed in 3.0
+         */
+        $adapterModifiers = [
+            '_light' => [
+                'skin' => 'light',
+            ],
+            '_slim' => [
+                'show_header' => false,
+            ],
+        ];
+
+        foreach ($listAdapters as $index => $adapter) {
+            foreach ($adapterModifiers as $key => $options) {
+                if (0 === \substr_compare($adapter, $key, -\strlen($key))) {
+                    $defaultAdapter = \str_replace($key, '', $adapter);
+                    $this->addAdapterOptionsToView($route, [$defaultAdapter => $options]);
+                    $listAdapters[$index] = $defaultAdapter;
+
+                    @\trigger_error(
+                        'The usage of the "' . $adapter . '" is deprecated.
+                                    Please use "' . $defaultAdapter . '"  with adapterOptions instead.',
```

Comment:
think this is formatted a bit strange. i think the string will contain a newline and a lot of spaces this way?